app/SmartAlbums/TagAlbum.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\SmartAlbums;
+
+use App\Photo;
+use Illuminate\Database\Eloquent\Builder;
+
+class TagAlbum extends SmartAlbum
+{
+	public function get_title()
+	{
+		return $this->title;
+	}
+
+	public function get_photos(): Builder
+	{
+		// /!\ this may reveal private picture.
```

Comment:
@ildyria Are you sure? I don't see the code doing anything different from Starred or Recent. In particular, `filter` is being applied at the end. We should test, of course...